_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -9,84 +9,297 @@ nav_order: 50
+      "request": {
+        "indices": ["system-metrics"],
+        "body": {
+          "size": 0,
+          "aggs": {
+            "avg_cpu_usage": {
+              "avg": {
+                "field": "cpu_usage"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "actions": [
+    {
+      "name": "Notify on Slack",
+      "type": "slack",
+      "slack": {
+        "message": "High CPU usage detected: {{ctx.payload.aggregations.avg_cpu_usage.value}}%"
+      }
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+**Low disk space alert**
+```json
+{
+  "name": "Low Disk Space Alert",
+  "description": "Alerts when the available disk space falls below 20%",
+  "schedule": {
+    "interval": {
+      "period": 30,
+      "unit": "MINUTES"
+    }
+  },
+  "trigger": {
+    "condition": {
+      "script": {
+        "source": "
+          double diskUsagePercentage = ctx.payload.aggregations.disk_usage_percentage.value;
+          return diskUsagePercentage > 80;
+        "
+      }
+    }
+  },
+  "input": {
+    "search": {
+      "request": {
+        "indices": ["system-metrics"],
+        "body": {
+          "size": 0,
+          "aggs": {
+            "disk_usage_percentage": {
+              "script": {
+                "source": "
+                  double totalDiskSpace = doc['disk_total'].value;
+                  double usedDiskSpace = doc['disk_used'].value;
+                  return (usedDiskSpace / totalDiskSpace) * 100;
+                "
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "actions": [
+    {
+      "name": "Notify on Email",
+      "type": "email",
+      "email": {
+        "to": ["ops@example.com"],
+        "subject": "Low Disk Space Alert",
+        "message": "Available disk space is below 20%: {{ctx.payload.aggregations.disk_usage_percentage.value}}%"
+      }
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+## Viewing alerting events
+
+To view alerting events, follow these steps:
+
+1. Open your local instance of OpenSearch Dashboards. This is typically `http://localhost:5601/app/home#/`.
+2. Navigate to **Alerting** from the main menu. You should see a list of all the alerting monitors you have configured. Select the monitor for which you want to view alerts.
+3. In the monitor details view, you'll see several tabs. Select the **Alerts** tab.  
+4. Explore events. The **Alerts** tab displays a list of all the alert instances that have been triggered for this monitor. Each alert shows details such as start and end time, status (Active, Acknowledged, Resolved), reason, and severity.
+5. Filter and search events based on specific criteria. For example, you can filter alerts by status using the filter dropdown at the top of the alerts table. 
+6. To view more details about a specific alter instance, select the alert in the table. This opens a flyout panel with additional information such as duration, context, history, and actions taken. 
```

Comment:
Is `alter` a typo here?

Reply:
Also, the flyout panel is only available from the monitor details for Per document monitors. The flyout is available for other monitor types from the alerting homepage. Here is a screenshot illustrating the kind of info that's displayed. It doesn't currently show things like "history, and actions taken."
![Screenshot 2024-05-29 at 11 43 10 AM](https://github.com/opensearch-project/documentation-website/assets/79280347/355303ac-ed12-4f6a-926d-843fa737d669)